src/js/main-window.js

```diff
@@ -6,9 +6,13 @@ var Accel = require('ui/accel');
 var Utils = require('utils');
 var Menu = require('menu');
 var Elements = require('elements');
+var Feature = require('platform/feature');
+var Settings = require('settings');
+
+Accel.init();
 
 var mainWindow = new UI.Window({
-  fullscreen: true, 
```

Comment:
Setting status to false has the same effect as setting fullscreen to true. You can see the usage of both properties on lines 265-270 in [https://github.com/pebble/pebblejs/blob/0a5871ca591dd6df71974b7c0a076728fca95414/src/js/ui/window.js](url)